### DISCUSSION TITLE
Best Practices for Remote OTBR Configuration via Cloud

### DISCUSSION THREAD

**Discussion by cedriczou13** [openthread/openthread#9467] [Q&A] [answered]
Hi everyone,

I'm exploring ways to remotely adjust OTBR features and retrieve its status via the cloud. What would be the most efficient implementation for this? Should I consider creating an external script outside of the OTBR software and send CLI commands, or is there a more integrated approach that's recommended?

**Comment by wgtdkp** [comment, marked as answer]
Check the d-bus API here: https://github.com/openthread/ot-br-posix/blob/main/src/dbus/server/introspect.xml
And the RESTful API here: https://github.com/openthread/ot-br-posix/blob/main/src/rest/openapi.yaml
